src/structures/MessageMentions.js

```diff
@@ -115,6 +116,22 @@ class MessageMentions {
     }
     return this._channels;
   }
+
+  /**
+   * Check if a user is mentioned. Takes into account user mentions, role
+   * mentions, and @everyone/@here mentions.
+   * @param {UserResolvable|GuildMember|Role|GuildChannel} data User/GuildMember/Role/Channel to check
+   * @param {boolean} [strict=true] If role mentions and everyone/here mentions should be included
+   * @returns {boolean}
+   */
+  has(data, strict = true) {
+    if (strict && this.everyone) return true;
+    if (strict && data instanceof GuildMember) {
+      for (const role of this.roles) if (data.roles.has(role.id)) return true;
+    }
+    const id = data.id ? data.id : data.user ? data.user.id : data;
```

Comment:
member has id too (as a getter), so this can be simplified to `const id = data.id || data;`